src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,53 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+/**
+ * Deadline task that takes in a description and date.
+ */
+public class Deadline extends Task {
+
+    private LocalDate date;
+
+    /**
+     * Constructor for creating a Deadline Task object.
+     *
+     * @param task The description of the task.
+     * @param date The LocalDate object that specifies the due date.
+     */
+    public Deadline(String task, LocalDate date) {
```

Comment:
The naming for task description would be better to be named taskDescription rather than just task.